app/api/migrations/migrations/36-populate-mimetype-on-attachments/index.js

```diff
@@ -0,0 +1,34 @@
+import request from 'shared/JSONRequest';
+import { attachmentsPath } from 'api/files/filesystem';
+import { execSync } from 'child_process';
+
+export default {
+  delta: 36,
+
+  name: 'populate-mimetype-to-attachment',
+
+  description: 'Populates mimetype of an attachment from a url ',
+
+  async up(db) {
+    process.stdout.write(`${this.name}...\r\n`);
+    const cursor = await db.collection('files').find({});
+
+    // eslint-disable-next-line no-await-in-loop
+    while (await cursor.hasNext()) {
+      // eslint-disable-next-line no-await-in-loop
+      const file = await cursor.next();
+      if (file.url && !file.mimetype) {
+        // eslint-disable-next-line no-await-in-loop
+        const response = await request.head(file.url);
+        const mimetype = response.headers.get('content-type') || undefined;
+        db.collection('files').updateOne({ _id: file._id }, { $set: { mimetype } });
+      }
+      if (file.filename && file.type === 'attachment' && !file.mimetype) {
+        const mimetype = execSync(`file --mime-type -b "${attachmentsPath(file.filename)}"`)
```

Comment:
Using the file command means that this migration is only going to be runnable in unix environments, making it imposible for windows-hosted (if any) or developers using Windows to upgrade Uwazi.
I think we should either use something that is cross-OS or provide a fallback in case the `file` util doesn't exist.
Let's discuss this with the team.

Reply:
I was avoiding having to install any packages generally, coz migrations should be as independent as possible, will look into whether its possible to have a windows solution without having to install dependencies.

Reply:
I have added https://www.npmjs.com/package/mime-types for mime types... it seems its actively being maintained